src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+
+**In the Beginning...**
+
+In 2002, a couple guys saw the potential of being able to watch media on their own terms, through what was a new type of device: a "connected" TV device. They wanted to bring many things you'd traditionally need a computer for to the living room, operating it from the couch without needing a keyboard or mouse, controlling events through what they called a *10 foot interface* because, well, it was an interface you could sit 10 feet away from.
+
+While special-purpose devices already existed, they usually were things like DVD players, and they weren't ‘connected’ to anything else or even had the ability to hook up to a network. Indeed, even the TiVo, one of the most impressive devices of the time, lacked an Ethernet port. One of the creators of Samba, an open source interoperability application to share files between Linux and Windows, eventually figured out how to add a network port to a TiVo; however, even though it was the first mass marketed consumer Linux-based appliance, it was still very much a closed, walled garden.
+
+The Xbox changed all that. In 2001, Microsoft released their first console, leveraging their work on x86 platforms to build out a version of Windows specialized for running games. The familiarity allowed game developers to easily port to the new gaming console; the fact that Microsoft sold them at a loss, knowing they would make it back in game sales, ensured rapid consumer adoption. Luckily for the hackers out there, it was more similar to than different from something the majority were familiar with: the standard personal computer running Windows. Very quickly, folks like Bunnie figured out ways around the security, achieving the holy grail in record time for a console: the ability to run untrusted code.
+
+Long before we had the Raspberry Pi, OpenWrt routers and other tinkerer's dream "open" environments, Xbox hacking quickly became the fastest route to general-purpose, connected computing. The availability of powerful, affordable, standardized hardware enabled an entire community of hackers wanting to break out of Microsoft’s walled garden and run whatever code they wanted on their Xbox. Emulation, or the ability to trick old games such as NES and Sega Genesis into believing the Xbox is the same hardware, was one of the first big use cases, since we now had something powerful enough that hooked up to a regular TV and offered a standard controller.
+
+**XBMC is Born**
+
+While many folk worked on getting Linux and various other things running, the project that attracted me the most was Xbox Media Player. This brought the ability to play compressed media to the Xbox: up until this point, only a computer could play these files, and if you were one of the those super early adopters who had network cables strung across your house, you could now even stream it over the network or the Internet (which was definitely not fast enough for video… yet!).
```

Comment:
Actually, the Dreamcast was hacked and had homebrew media players a year earlier, but probably not worth an edit.